examples/platform/esp32/Bluetooth.h

```diff
@@ -0,0 +1,47 @@
+/*
+ *
```

Comment:
I am wondering if this functionality should go in `src/platform/ESP32/nimble/BLEManagerImpl.cpp`. Thoughts?